### PR TITLE
ParserBolt uses right Status class + relies on standard metrics

### PR DESCRIPTION
**Pull request by jnioche**
Implements #64 
Could you please review?

### CODE REVIEW

File: src/main/java/com/digitalpebble/storm/crawler/bolt/ParserBolt.java [hunk 1]
```diff
@@ -152,21 +148,15 @@ public void prepare(Map conf, TopologyContext context,
...
 
-        this.eventMeters = context.registerMetric("parser-meter",
-                new MeterMetric(), 5);
-        this.eventTimers = context.registerMetric("parser-timer",
-                new TimerMetric(), 5);
-        this.eventHistograms = context.registerMetric("parser-histograms",
-                new HistogramMetric(), 5);
-
+        this.eventCounter = context.registerMetric(this.getClass()
+                .getSimpleName(), new MultiCountMetric(), 10);
```
Comment (GuiForget) [inline src/main/java/com/digitalpebble/storm/crawler/bolt/ParserBolt.java]: How about using the component name instead of the class name?

**Comment by GuiForget** [comment]
Looks good beside my quick comment on the metric's name

File: src/main/java/com/digitalpebble/storm/crawler/bolt/ParserBolt.java (hunk 1 above)
Comment (jnioche) [inline src/main/java/com/digitalpebble/storm/crawler/bolt/ParserBolt.java]: The component ID is already handled by the metrics mechanism and will be set for the datapoints.  Specifying the class here allows to add more info on which class was actually used, e.g. various flavours of parsers all running as 'parse' component.

Here is what you'd get by using the component name. Bit redundant, isn't it?

```
Mint15-Vostro-410:1027	  6:parse      	parse                  	{outlink_kept=952, tuple_in=14, error_out_link_parsing_MalformedURLException=105, tuple_success=14, outlink_filtered=119, outlink_outsideSourceDomainOrHostname=217}
```

**Comment by jnioche** [comment]
thanks for reviewing it @GuiForget

**Pull request merged**